usr/bin/NETMODE

```diff
@@ -5,13 +5,35 @@ function show_usage() {
     echo ""
 }
 
+function configure_client() {
+    /etc/init.d/odhcpd stop
+    uci set network.lan.proto='dhcp'
```

Comment:
`uci commit` is required after a `uci set`

Reply:
you might want to read the file instead of just the diff. both the original and my version call uci commit on the second to last line.  I just made the calls functions to avoid code duplication

Reply:
Ah, I just missed it at the bottom!